### ISSUE TITLE
Improve Copy of Site Kit CTA on WP Tools page

### ISSUE THREAD

**Issue by wpdarren** [google/site-kit-wp#4379]
## Bug Description

The copy could be improved on the Reset Site Kit CTA in Tools > Available Tools (`/wp-admin/tools.php`)

Currently it reads: `Resetting will disconnect all users and remove all Site Kit settings and data within WordPress. You and any other users who wish to use Site Kit will need to reconnect to restore access.`.  

Maybe we could make a small change by removing one of the and's.  Change to:

`Resetting will disconnect all users, remove all Site Kit settings and data within WordPress.  You and any other users who wish to use Site Kit will need to reconnect to restore access. `

Not a huge issue but one that jumped out at me. 

![image](https://user-images.githubusercontent.com/73545194/142654393-bebc68d3-22e6-45e1-ab85-442a40910715.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
Maybe we can enhance the wording but it doesn't read too badly to me 😄

**Comment by aaemnnosttv** [comment]
After discussing with @felixarntz we decided it was preferable to keep it as-is.

The wording here was chosen rather carefully (see #1825 for more context) and is the same language we use for the reset dialog shown everywhere else resetting Site Kit is possible.

As to the suggested change, the proposed wording reads more like a list with only two elements which feels like an item is missing.  We discussed some potential variations on this but ultimately decided to leave it unchanged for now. Thanks for the suggestion!
